utils/frame/frame-utilities-cli/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+name = "frame-utilities-cli"
+version = "2.0.0-rc2"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "CLI utils"
+documentation = "https://docs.rs/frame-utils"
+
+[dependencies]
+sp-core = { version = "2.0.0-rc2", path = "../../../primitives/core" }
+sc-cli = { version = "0.8.0-rc2", path = "../../../client/cli" }
+sp-runtime = { version = "2.0.0-rc2", path = "../../../primitives/runtime" }
+structopt = "0.3.8"
+codec = { version = "1.3.0", package = "parity-scale-codec" }
+pallet-balances = { version = "2.0.0-rc2", path = "../../../frame/balances" }
+pallet-indices = { version = "2.0.0-rc2", path = "../../../frame/indices" }
+frame-utils = { path = "../../frame-utils", version = "2.0.0-rc2" }
+hex = "0.4.2"
+serde = "1.0.106"
+hyper = "0.12.35"
+libp2p = "0.18.1"
+bip39 = "0.6.0-beta.1"
+jsonrpc-core-client = { version = "14.0.3", features = ["http"] }
+sc-rpc = { version = "2.0.0-rc2", path = "../../../client/rpc" }
+sc-service = { version = "0.8.0-rc2", default-features = false, path = "../../../client/service" }
+sc-client-api = { version = "2.0.0-rc2", default-features = false, path = "../../../client/api" }
+futures01 = { package = "futures", version = "0.1.29" }
+futures = "0.3"
+
+[dev-dependencies]
+tempfile = "3.1.0"
+sp-io = { version = "2.0.0-rc2", path = "../../../primitives/io" }
+frame-system = { version = "2.0.0-rc2", path = "../../../frame/system" }
+
+[features]
+balances = []
```

I'd expect all pallets-dependencies to be optionally activatable through feature flags.
```suggestion
default = []
balances = ["pallet-balances"]
indices = ["pallet-indices"]
```